From 3bce6332769b3f2191e7746b6bf57b9381d7dd47 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Sun, 15 Sep 2024 21:24:25 +0800
Subject: [PATCH 07/10] FROM ARCHLINUX: fix: python 3.12 compatibility

---
 .../Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py b/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py
index 693e01e088e0..35a7c9528ab5 100755
--- a/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py
+++ b/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py
@@ -15,7 +15,7 @@ UPT
 #
 import locale
 import sys
-from imp import reload
+from impportlib import reload
 encoding = locale.getdefaultlocale()[1]
 if encoding:
     reload(sys)
-- 
2.46.2

